tests/sentry/workflow_engine/endpoints/test_validators.py

```diff
@@ -546,3 +547,120 @@ class MockConditionGroupValidator(BaseDataConditionGroupValidator):
 class MockDetectorValidator(BaseDetectorTypeValidator):
     data_source = MockDataSourceValidator()
     condition_group = MockConditionGroupValidator()
+
+
+class MockActionHandler(ActionHandler):
+    config_schema = {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "description": "The configuration schema for a Action Configuration",
```

Comment:
nit on both the descriptions here - maybe "The configuration schema for Action configuration" and "The configuration schema for Action data"

Reply:
I'll do this on a separate PR so i don't need to block on CI again.